src/main/javascript/service/main-menu-service.js

```diff
@@ -97,7 +97,7 @@ function Menu(menuId, options) {
             } else {
                 return [group, group.getEntries()];
             }
-        })]);
+        })], !!_.flattenDeep);
```

Comment:
as you can see here https://github.com/lodash/lodash/wiki/Changelog#v300 flatten was deep in 2.x so in both versions it will be the same result this way.